src/test/java/uk/gov/hmcts/probate/model/ccd/raw/request/CaseDataTest.java

```diff
@@ -460,6 +460,12 @@ public void shouldReturnDODFormattedWithST() {
         assertEquals("1st January 2000", caseData.getDeceasedDateOfDeathFormatted());
     }
 
+    @Test
+    public void shouldReturnDODFormattedWithSTPublic() {
+        CaseData caseData = CaseData.builder().build();
+        assertEquals("1st January 2000", caseData.convertDate(LOCAL_DATE));
+    }
+
```

Comment:
did there need to be any other tests for this? e.g. nd, rd etc. ?

Reply:
Not really, I am just testing whether setting the convertDate method to "public". My bad for adding "ST" to the test name.